src/utils/uri.ts

```diff
@@ -122,18 +122,22 @@ export type SrvConnectOptions = Omit<ConnectOptions, "servers"> & {
 
 export function parseSrvUrl(url: string): SrvConnectOptions {
   const data = parse_url(url);
+
+  const path = (data.pathname && (data.pathname.length > 1))
```

Comment:
I think the variable name `path` is not very appropriate

Reply:
What's your suggestion?
path of `mongodb://a:b@localhost:27017/dbName?authSource=admin2` uri is `/dbName`. That makes sense to 
![Screenshot from 2021-09-22 18-20-41](https://user-images.githubusercontent.com/47688578/134367459-a9693c8f-10bb-4b3f-be7e-8fc692301d7f.png)
me

Reply:
Maybe pathExists if I understood correctly that it's a boolean.

Reply:
It's not boolean, it's string

Reply:
Sorry github just cut the code sample after line 126

Reply:
Although this part of the uri is called `path`, it is in the connection string protocol of mongodb. Called `defaultAuthDB`
https://docs.mongodb.com/manual/reference/connection-string/#std-label-connections-standard-connection-string-format